helm/dagster/templates/helpers/_helpers.tpl

```diff
@@ -43,6 +43,7 @@ If release name contains chart name it will be used as a full name.
 dagit -h 0.0.0.0 -p {{ .Values.dagit.service.port }}
 {{- if $userDeployments.enabled }} -w /dagster-workspace/workspace.yaml {{- end -}}
 {{- with .Values.dagit.dbStatementTimeout }} --db-statement-timeout {{ . }} {{- end -}}
+{{- with .Values.dagit.logLevel }} --log-level {{ . }} {{- end -}}
```

Comment:
just to ensure - this with block does not eval if the value is unset? I believe the test above ensures that